docs/content/docs/dev/table/catalogs.md

```diff
@@ -738,3 +738,192 @@ Flink SQL> show tables;
 ```
 {{< /tab >}}
 {{< /tabs >}}
+
+## Catalog Store
+
+Catalog Store is used to store the configuration of catalogs. When using Catalog Store, the configurations
+of catalogs created in the session will be persisted in the corresponding external system of Catalog Store.
```

Comment:
What's `session` here? Besides sql-gateway, there's no session in flink. The session here is a little strange

Reply:
I think `session` represent a execution environment, not only in sql-gateway.  Such as session is used in bellow doc. 

https://github.com/apache/flink/blob/fc2b5d8f53a41695117f6eaf4c798cc183cf1e36/docs/content/docs/dev/table/catalogs.md?plain=1#L45

https://github.com/apache/flink/blob/fc2b5d8f53a41695117f6eaf4c798cc183cf1e36/docs/content/docs/dev/table/timezone.md?plain=1#L29